actionpack/lib/action_controller/metal.rb

```diff
@@ -142,9 +142,23 @@ def controller_name
       self.class.controller_name
     end
 
-    attr_internal :response, :request
+    ##
+    # Returns the ActionDispatch::Response object for the current request
+    attr_internal :response
```

Comment:
This is currently being documented as a method, but we could also mark it as an attribute which may be more accurate?
```suggestion
    ##
    # :attr_accessor: response
    # Returns the ActionDispatch::Response object for the current request
    attr_internal :response
```

Reply:
Yes, perhaps so.  The downside is that attributes don't show up when searching in the rendered docs sidebar.  Also, attributes are linkable in the sense that references to them will become links, but attributes aren't rendered with destination anchors, so the links just go to the top of the page.  I'm not sure whether those are RDoc issues or SDoc issues, though.